src/UI/examples/Button/Standard/with_js_binding.php

```diff
@@ -0,0 +1,14 @@
+<?php
+function with_js_binding() {
+    global $DIC;
+    $f = $DIC->ui()->factory();
+    $renderer = $DIC->ui()->renderer();
+
+    return $renderer->render(
+		$f->button()->standard("Goto ILIAS", "#")
```

Comment:
Hmm, your indeed correct.

So we are left with two options:
1) making the action optional to the constructor, which means loosing the possibility to enforce the "every XYZ has an action" constraint for the moment
2) requiring the user two fix the href manually in the JS

1) would also mean that we'd have to touch the places with actions we'll touch again in foreseeable future. 2) is a clear inconvenience. Your choice =)